### PR TITLE
[REVIEW] fix empty column issues with java memory usage API

### PR DESCRIPTION
**Pull request by revans2**
The java memory usage API has a few issues when little or no memory is being used.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -108,7 +108,7 @@
 - PR #3274 ORC writer: fix integer RLEv2 mode2 unsigned base value encoding
 - PR #3279 Fix shutdown hang issues with pinned memory pool init executor
 - PR #3280 Invalid children check in mutable_column_device_view
-
+- PR #3289 fix java memory usage API for empty columns
```
Comment (jlowe) [inline CHANGELOG.md]: Whitespace line between releases was deleted here.

**Comment by revans2** [comment]
rerun tests

**Comment by jlowe** [comment]
rerun tests

**Comment by revans2** [comment]
rerun tests

**Pull request merged**